### PR TITLE
Build: rclone retries when uploading artifacts

### PR DESCRIPTION
**Pull request by humitos**
Related https://github.com/readthedocs/readthedocs.org/issues/9947

### CODE REVIEW

File: readthedocs/storage/rclone.py [hunk 1]
```diff
@@ -44,6 +44,11 @@ class BaseRClone:
         # Skip based on checksum (if available) & size, not mod-time & size.
         "--checksum",
         "--verbose",
+        # Retry some times before failing
+        # (3 is the default, but making it explicit here)
```
Comment (ericholscher) [inline readthedocs/storage/rclone.py:48]: ❤️

**Review by ericholscher — APPROVED** [review]
Explicit is good!

File: readthedocs/storage/rclone.py [hunk 2]
```diff
@@ -44,6 +44,11 @@ class BaseRClone:
...
+        "--retries=3",
```
Comment (humitos) [inline readthedocs/storage/rclone.py:49]:
```suggestion
        "--retries=3s",
```

File: readthedocs/storage/rclone.py [hunk 3]
```diff
@@ -44,6 +44,11 @@ class BaseRClone:
...
+        # Wait 1 second between each retry
+        "--retries-sleep=1",
```
Comment (humitos) [inline readthedocs/storage/rclone.py:51]:
```suggestion
        "--retries-sleep=1s",
```

**Pull request merged**